_posts/blog/2017-08-21-google-assistant-development-for-android-devs-part-2.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Google Assistant Development for Android Devs (Part 2)"
+author: Patrick Jackson
+categories: [Development, Android]
+tags: [Google Assistant, Actions on Google, conversational interfaces, Kotlin]
+---
+
+
+![](https://storage.googleapis.com/kotlin-actions-sdk.appspot.com/actions-kotlin-java.png)
+
+This is part 2 in a series on developing Actions on Google.  [Part one is here](http://ticketmastermobilestudio.com/blog/google-assistant-development-for-android-devs-part-1).
+
+In my last post, I introduced an open source SDK for Actions on Google for Kotlin and Java.  Today, that library port is complete, with all features supported.  
+
+ - API.AI support
+ - Actions SDK support
+ - All tests ported and passing (~230)
+ - Conversation components example for API.AI & Actions SDK
+ - Transaction Sample for API.AI
+ - Java conversation sample
+ 
```

Comment:
I forgot about this, but Jekyll automatically sets the excerpt to be the first paragraph, which in this case is just the image. It might be good to include a bit more before the break, maybe here after the bullet list? You just have to add this bit:

```html
<!--more-->
```